src/index.html

```diff
@@ -5,9 +5,63 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <header></header>
+    <main>
+      <article class="card" data-qa="card">
+        <img
+        class="card__img"
+        src="./images/imac.jpeg"
+        alt="product image">
+
+        <h2 class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__review-box">
+          <div class="stars">
+            <div class="stars__star stars__star--1" href="#">
+            </div>
+
+            <div class="stars__star stars__star--2" href="#">
+            </div>
+
+            <div class="stars__star stars__star--3" href="#">
+            </div>
+
+            <div class="stars__star stars__star--4" href="#">
+            </div>
+
+            <div class="stars__star stars__star--5" href="#">
+            </div>
+          </div>
```

Comment:
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>

            <div class="stars__star"></div>

            <div class="stars__star"></div>

            <div class="stars__star"></div>

            <div class="stars__star></div>
          </div>
```
remove redundant href attribute for div tag
responsible for amount of active stars should be modifier for stars block as in task `stars`
![Screenshot from 2023-06-22 13-27-46](https://github.com/mate-academy/layout_product-cards/assets/33029864/7d1b56dd-792d-4dc4-9b54-a6ec440a3eb5)